### PR TITLE
Version 1.0.0 rc1 updates

### PR DESCRIPTION
**Pull request by arscan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,19 +24,43 @@ Docker is the recommended installation method for Windows devices and can also b
...
+## Supported Browsers
+
+Inferno has been tested on the latest versions of Chrome, Firefox, Safari, and Edge.  Internet Explorer is not supported at this time.
+
 ## Unit Tests
 
-The *Inferno SMART on FHIR Test App* contains a robust set of self-tests to ensure that the
+Inferno contains a robust set of self-tests to ensure that the
 test clients conform to the specification and performs as intended.  To run these tests, execute the following
 command:
```
Comment (notpace) [inline README.md]: Maybe we should point to the quick start guide here instead.

File: lib/app/sequences/03_patient_standalone_launch_sequence.rb [hunk 2]
```diff
@@ -0,0 +1,237 @@
+module Inferno
+  module Sequence
+    class PatientStandaloneLaunchSequence < SequenceBase
```
Comment (notpace) [inline lib/app/sequences/03_patient_standalone_launch_sequence.rb]: Remove 'Patient'?

File: lib/app/sequences/04_provider_ehr_launch_sequence.rb [hunk 3]
```diff
@@ -0,0 +1,257 @@
+module Inferno
+  module Sequence
+    class ProviderEHRLaunchSequence < SequenceBase
```
Comment (notpace) [inline lib/app/sequences/04_provider_ehr_launch_sequence.rb]: Remove 'Provider'?

File: README.md [hunk 4]
```diff
@@ -1,32 +1,16 @@
-# Inferno
+![Inferno Logo](https://github.com/siteadmin/inferno/blob/master/public/images/inferno_logo.png "Inferno Logo")
```
Comment (notpace) [inline README.md]: This image is broken

File: README.md (hunk 1 above)
Comment (notpace) [reply README.md]: Not for the Unit tests section, but for the `Basic Usage Instructions`

File: README.md (hunk 4 above)
Comment (arscan) [reply README.md]: Broken because it doesn't exist on master yet.

File: lib/app/sequences/04_provider_ehr_launch_sequence.rb (hunk 3 above)
Comment (arscan) [reply lib/app/sequences/04_provider_ehr_launch_sequence.rb]: I'll do this after.  This needs more testing and isn't visible on the web interface.

File: lib/app/sequences/03_patient_standalone_launch_sequence.rb (hunk 2 above)
Comment (arscan) [reply lib/app/sequences/03_patient_standalone_launch_sequence.rb]: I'll do this after. This needs more testing and isn't visible on the web interface.

**Pull request merged**